psi4/driver/qcdb/libmintsmolecule.py

```diff
@@ -2984,17 +2984,22 @@ def symmetry_from_input(self):
         """
         return self.PYsymmetry_from_input
 
-    def symmetrize(self):
+    def symmetrize(self, tol=None):
```

Comment:
could [either of these](https://github.com/psi-rking/psi4/blob/4fa13d21af17adf4351f6f804a3a43d94833e224/psi4/driver/qcdb/libmintsmolecule.py#L53-L54) be used? Then you wouldn't have to have the if/else and just always `atom_map = compute_atom_map(self, tol=tol)`.

Reply:
Not without some reorganization.  Presently, symmetrize calls compute_atom_map() which is hardwired to 0.05!  Not only hardwired, but a long way from 10^-8.  The challenge is these functions are getting used for a number of different purposes.  It may be hard not to break something if we change the default behavior.  The 10^-8 is relevant for things like telling us what the input point group is.  If one is symmetrizing a gradient, then you might only expect 9th decimal rounding.  On the other hand, if you are symmetrizing a geometry step then you might accept even 0.1.  It's hard to choose a reasonable default unless you know the purpose.  (Furthermore, compute_atom_map currently raises an exception, when for some purposes this isn't desirable.)

Reply:
Of course compute_atom_map (0.05) is getting used in many places including the construction of cdsalcs - everything that gets symmetry adapted, I presume.  Perhaps QCDB can break free of this legacy.  But I fear it's a can of worms that might not be worth opening.